### ISSUE TITLE
retreving the the last layer values to train it with SVM

### ISSUE THREAD

**Issue by vinayakumarr** [fchollet/keras#6178]
My model is give below
# 1. define the network
model = Sequential()
model.add(LSTM(64,input_dim=50))  # try using a GRU instead, for fun
model.add(Dropout(0.1))
model.add(Dense(4))
model.add(Activation('softmax'))
model.load_weights("logs/1layer/checkpoint-9957.hdf5")

I have trained a lstm network and stored the checkpoints and in testing I am loading the checkpoints to evaluate. So now i want to get the features for the train and test data to give those features to the LSTM network. Could you please tell how to do this? This is a kind request

**Comment by sallamander** [comment]
Hey @vinayakumarr, have you seen [this section](https://keras.io/getting-started/faq/#how-can-i-obtain-the-output-of-an-intermediate-layer) of the FAQ?
